src/main/java/dk/bec/polonez/reservationsystem/service/ReservationService.java

```diff
@@ -74,7 +82,7 @@ public ResponseReservationDto addReservation(CreateReservationDto reservationDto
     }
 
     public ResponseReservationDto updateReservation(UpdateReservationDto reservationDto) {
-        Reservation reservationExistingTest = getById(reservationDto.getId());
+        Reservation existingReservation = getById(reservationDto.getId());
 
         Reservation.ReservationBuilder reservationBuilder = Reservation.builder();
```

Comment:
nie ma potrzeby tworzyc tej zmiennej

co wiecej, bede wstanie zupdatowac rezerwacje, tak, ze bedzie sie zaczynac 01.01.2020 a konczyc 01.01.1990 ?
czy w kazdym statusie mozna updatowac rezerwacje? np. czy rezerwacje z przeszlosci mozna updatowac? 
czy moge zupdatowac rezerwacje tak, ze bedzie sie pokrywac z inna rezerwacja? 
moze czegos nie dostrzeglem, ale obecnie wydaje sie, ze co przesle uzytkownik to jest zapisane

Reply:
Nie pisałem tego kodu, ale dam znać autorowi ;)